interface/themes/style_light.css

```diff
@@ -470,6 +470,10 @@ a:hover {
   margin: 5px;
 }
 
+#pnotes_ps_expand  div.tab table td {
+  padding-right: 10px;
+}
```

Comment:
Is this something that needs to go into every theme?
(note there are 20+ of these now, and I'm happy to take on that glorious task, if needed. Just let me know.)

Reply:
I think it should go in every theme, and would be **overjoyed** to hand this over to you :)